docs/src/man/split_apply_combine.md

```diff
@@ -778,3 +885,338 @@ julia> df
    5 │     2  missing      5
    6 │     3  missing      6
 ```
+
+## Context dependent expressions
+
+Operation specification language supports the following context dependent
+operations:
+
+* getting the number of rows (`nrow`);
+* getting the proportion of rows (`proprow`);
+* getting the group number (`groupindices`);
+* getting a vector of group indices (`eachindex`).
+
+These operations are context dependent, because they do not require input column
+name in the operation specification syntax.
+
+These four exceptions to the standard operation specification syntax were
+introduced for user convenience as these operations are often needed in
+practice.
+
+Below each of them is explained by example.
+
+First create a data frame we will work with:
+
+```jldoctest sac
+julia> df = DataFrame(customer_id=["a", "b", "b", "b", "c", "c"],
+                      transaction_id=[12, 15, 19, 17, 13, 11],
+                      volume=[2, 3, 1, 4, 5, 9])
```

Comment:
The data would be too big and I was afraid that it would not be 100% clear what is going on. In this example all data is displayed fully in the output.